conference/ical.py

```diff
@@ -34,20 +34,20 @@ def encode(line):
 def content(name, value, params=None):
     if params:
         t = []
-        for pname, pvalue in params.items():
+        for pname, pvalue in list(params.items()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;